examples/generative/pixelcnn.py

```diff
@@ -163,8 +160,8 @@ def call(self, inputs):
             probs = pixel_cnn.predict(pixels)[:, row, col, channel]
             # Use the probabilities to pick pixel values and append the values to the image
             # frame.
-            pixels[:, row, col, channel] = tf.math.ceil(
-                probs - tf.random.uniform(probs.shape)
+            pixels[:, row, col, channel] = ops.ceil(
+                probs - keras.random.uniform(probs.shape)
```

Comment:
Random note -- the line above will only work eagerly in JAX (which is the case here). For JAX compiled execution we'd need to pass a `SeedGenerator` to the `random.uniform` call.